docs/site/BelongsTo-relation.md

```diff
@@ -336,12 +343,18 @@ belongs to a `Customer`.
 
 After setting up the relation in the repository class, the inclusion resolver
 allows users to retrieve all orders along with their related customers through
-the following code:
+the following code in repository level:
 
 ```ts
 orderRepo.find({include: [{relation: 'customer'}]});
 ```
 
+or use APIs with controllers:
+
+```
+GET http://localhost:3000/orders?filter[include][][relation]=customer
```

Comment:
is it an extra `[]`? I am looking at the similar include filter in LB3 https://loopback.io/doc/en/lb2/Include-filter.html#rest-examples:

```
/customers?filter[include][reviews]=author
```

Reply:
No, `/customers?filter[include][reviews]=author` wouldn't work because `include.filter` is not a function.

And different queries between LB3 and LB4 make me wonder if we need a `querying data` page for LB4 as well. Cause right now the only way to check the query syntax is to go to the LB3 site.

Reply:
The include filter for LB2 is different from the LB4 one. In LB4 it's `{include: [{relation: 'customer'}]}`, whereas in LB2 (from my understanding) it's `{include: {customer: [/* ... */]}}`.